4001-5000/LIT4723TarikaWaldaAmid.xml

```diff
@@ -4,14 +4,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="t1">ታሪከ፡ ወልደ፡ ዐሚድ፡</title>
-                <title xml:lang="gez" corresp="#t1" type="normalized">Tārika Walda ʿAmid</title>
-                <title xml:lang="en" corresp="#t1">History of Walda ʿAmid</title>
-                <title xml:lang="ar" xml:id="t2">المجمع المبارك</title>
-                <title xml:lang="ar" corresp="#t2" type="normalized">al-Maǧmaʿ al-mubārak</title>
-                <title xml:lang="en" corresp="#t2">The Blessed Collection</title>
-                <title xml:lang="en" xml:id="t3">History</title>
-                <title xml:lang="en" xml:id="t4">Universal Chronicle</title>
+                <title xml:lang="gez" xml:id="t1">ታሪከ፡ ወልደ፡ አሚድ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Tārika Walda ʾAmid</title>
```

Comment:
I would add (general record) in the title and also mention explicitly in the abstract that this is the general record to be used in cataloguing only when the recension is not known